components/brave_rewards/core/state/state.cc

```diff
@@ -346,9 +346,6 @@ std::vector<double> State::GetAutoContributeChoices() {
   if (!base::Contains(amounts, current_amount)) {
     amounts.push_back(current_amount);
     std::sort(amounts.begin(), amounts.end());
-
-    ledger_->SetState(kParametersAutoContributeChoices,
```

Comment:
This function should just be:
```
const std::string amounts_string =
    ledger_->GetState<std::string>(kParametersAutoContributeChoices);
return StringToVectorDouble(amounts_string);
```
right?

Reply:
This function also adds in the value that doesn't currently exist in the list. Although we probably haven't hit that case, and may not, I'd prefer not to change that aspect here.